common/src/main/java/com/example/android/uamp/media/PackageValidator.kt

```diff
@@ -132,6 +132,12 @@ class PackageValidator(context: Context, @XmlRes xmlResId: Int) {
              * with apps such as Wear OS.
              */
             callerPackageInfo.permissions.contains(BIND_NOTIFICATION_LISTENER_SERVICE) -> true
+            /**
+             * Whitelist the com.example.* package name for MCT.
+             * https://github.com/googlesamples/android-media-controller.
+             * com.example.* apps cannot be distributed on Google Play.
```

Comment:
People can still sideload APKs (outside of Google Play's distribution).
Since a lot of people tend to copy-paste code from this repository, I'd be a bit conservative with this whitelist.

Do you think wrapping this entry around a `if (BuildConfig.DEBUG)` and hardcoding the full package (`com.example.android.mediacontroller`) would still fulfil your testing use-case while still keeping us safe enough for production builds?

Reply:
I think I'd like to avoid listing specific package names in here.

I'd be fine adding a condition to allow the connection if the caller is signed with the same signing cert as UAMP. It seems like that's probably pretty likely in debug situations.

Otherwise, I think developers can either list the package signature themselves, or just let anyone connect by skipping this check entirely.